src/utilities.ts

```diff
@@ -44,6 +44,11 @@ export function getCurrentYear(): number {
   return new Date().getFullYear();
 }
 
+// the number of years +- the current year to show in new semester and onboarding dropdowns
+export function getYearRange(): number {
+  return 6;
+}
```

Comment:
What does "the number of years +- the current year" mean?

Also maybe it should be a constant instead of a function for now?

Reply:
Will clarify the comment and make it a constant, thank you!